src/main/java/com/softserveinc/dokazovi/controller/PostController.java

```diff
@@ -66,4 +67,18 @@
 				.status(HttpStatus.OK)
 				.body(postService.findAllByMainDirection(direction, type, tags, PostStatus.PUBLISHED, pageable));
 	}
+
+	@ApiPageable
+	@ApiOperation(value = "Find latest posts by some expert")
+	@GetMapping(POST_LATEST_BY_EXPERT)
+	public ResponseEntity<Page<PostDTO>> findLatestByExpert(
+			@PageableDefault(size = 9, sort = {"createdAt", "id"}, direction = Sort.Direction.DESC) Pageable pageable,
+			@ApiParam(value = "Expert's id")
+			@RequestParam Integer expert,
+			@ApiParam(value = "Post type id")
+			@RequestParam(required = false) Integer type) {
```

Comment:
Maybe it would be better to accept Set of Ids for type as we can filter posts by more than one type, according to the mocks.